includes/discovery/sensors/current/eltex-mes21xx.inc.php

```diff
@@ -1,6 +1,6 @@
 <?php
```

Comment:
It looks like these can be done in yaml, but that we can leave for another day™

Reply:
sorry @Jellyfrog  but no, it could not be done in YAML
OIDs are not ending with Index
example:
as you could see, index (57, 58) is in the middle

rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew1 = 8
rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew2 = 0
rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew3 = 0
rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew4 = 0
rlPhyTestGetResult.58.rlPhyTestTableTransceiverTemp = 43
rlPhyTestGetResult.58.rlPhyTestTableTransceiverSupply = 3338400
rlPhyTestGetResult.58.rlPhyTestTableTxBias = 11150
rlPhyTestGetResult.58.rlPhyTestTableTxOutput = -6068
rlPhyTestGetResult.58.rlPhyTestTableRxOpticalPower = -7129
rlPhyTestGetResult.58.rlPhyTestTableDataReady = 1
rlPhyTestGetResult.58.rlPhyTestTableLOS = 0
rlPhyTestGetResult.58.rlPhyTestTableTxFault = 0

Reply:
Why follow a standard when you can ignore it